Symfony/CS/Fixer/Contrib/SelfFixer.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ */
+class SelfFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        $elements = $this->findClassyElements($tokens);
+
+        foreach ($elements as $element) {
+            $this->replaceNameOccurrences($tokens, $element);
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'Inside a classy element "self" should be preferred to the class name itself';
+    }
+
+    private function findClassyElements(Tokens $tokens)
```

Comment:
what is the purpose of that method? what is `classy` ?
please provide a documentation.